pkg/image/gce_test.go

```diff
@@ -0,0 +1,33 @@
+package image
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/osbuild/images/pkg/manifest"
+	"github.com/osbuild/images/pkg/osbuild"
+	"github.com/osbuild/images/pkg/rpmmd"
+	"github.com/osbuild/images/pkg/runner"
+)
+
+func TestNewGCETarPipeline(t *testing.T) {
+	var repos []rpmmd.RepoConfig
+	m := &manifest.Manifest{}
+	runner := &runner.Fedora{}
+
+	buildPipeline := manifest.NewBuild(m, runner, repos, nil)
+	buildPipeline.Checkpoint()
+
+	imgPipeline := manifest.NewRawImage(buildPipeline, nil)
+	imgPipeline.SetFilename("foo.img")
```

Comment:
May as well also have a test for `disk.raw` just to be sure.
Looks good to me though, other than the merge conflict :)

Reply:
Thanks, excellent point. I updated this now to test both cases (transform and no-transform).